Source/WebCore/testing/Internals.cpp

```diff
@@ -3203,7 +3209,13 @@ String Internals::serviceWorkerClientInternalIdentifier(const Document& document
 
 RefPtr<WindowProxy> Internals::openDummyInspectorFrontend(const String& url)
 {
-    auto* inspectedPage = contextDocument()->frame()->page();
+    if (!contextDocument())
+        return nullptr;
+
+    auto* inspectedPage = contextDocument()->page();
```

Comment:
I’d think this would need to be `RefPtr` instead of `auto*` in our new safer C++ model.

Reply:
OK, I changed this and other refcounted objects in this method (except frontendWindowProxy since WindowProxy::window() is trivial).